chia/_tests/core/data_layer/test_data_store.py

```diff
@@ -1127,28 +961,27 @@ async def test_kv_diff(data_store: DataStore, store_id: bytes32) -> None:
     insertions = 0
     expected_diff: set[DiffData] = set()
     root_start = None
+    keys: list[bytes] = []
+
     for i in range(500):
         key = (i + 100).to_bytes(4, byteorder="big")
         value = (i + 200).to_bytes(4, byteorder="big")
-        seed = leaf_hash(key=key, value=value)
-        node_hash = await data_store.get_terminal_node_for_seed(store_id, seed)
+
         if random.randint(0, 4) > 0 or insertions < 10:
             insertions += 1
-            side = None if node_hash is None else data_store.get_side_for_seed(seed)
-
-            await data_store.insert(
+            await data_store.autoinsert(
```

Comment:
Does this test pattern need to change or can it stay the same?  Or maybe there's another reason to change it?